doc/user-guide.adoc

```diff
@@ -347,6 +348,27 @@ spec:
             storage: 1Gi
 ----
 
+=== Service ports
```

Comment:
Thanks @JusteenR, some comments:

- This section perhaps could be moved above Basic storage section, as typically users would configure service ports prior to storage. 
- It'll be good to specify the parameters that are associated with the primary port: `service.port`, `service.portName`, `service.targetPort` and `service.nodePort`.
- Also, specify that primary port is exposed from the application container. It is also used to configure Route (or Ingress), Service binding and Knative service.
- It's possible to specify an alternate port for Service Monitor using the `monitoring.endpoints` parameter.